src/components/Button/Button.module.less

```diff
@@ -34,30 +34,30 @@
     }
 
     &--primary {
-        background-color: @primary_color;
-        color: white;
+        background-color: var(--reactist-button-primary-fill);
+        color: var(--reactist-button-content-primary);
```

Comment:
Although I'm a fan, switching to CSS custom properties will have the consequence of Todoist not being able to adopt this, or having to style them fully instead of using the theme feature. This because Todoist still supports IE11 which does not have support for CSS custom properties.

Reply:
Hmm, I'm not sure what to do here. Without CSS variables, styling becomes much harder! Do we have a timeline on when we can retire IE 11 support? cc @henningmu